staging/src/k8s.io/client-go/util/flowcontrol/backoff.go

```diff
@@ -49,23 +48,19 @@ type Backoff struct {
 	maxJitterFactor float64
 }
 
-func NewFakeBackOff(initial, max time.Duration, tc *testingclock.FakeClock) *Backoff {
```

Comment:
Even though it bothers me to keep the current name while taking an interface, doing so would preserve compatibility with existing callers. If we're going to try to improve go signature compatibility, I would probably switch this to `NewFakeBackOff(initial, max time.Duration, tc clock.Clock) *Backoff` rather than renaming the method

That preserves compatibility with all existing callers, at least

Reply:
yeah,